### PR TITLE
Features/create join us

### PR DESCRIPTION
**Pull request by PooriaT**
Join Us based on the [Figma design](https://www.figma.com/file/RTNxJMR0l59uOQJDbGNeTc/OpenBC-Redesign?type=design&node-id=111-783&mode=design&t=6pRT0s2tgw0TmuUs-0)

### CODE REVIEW

**Review by Unknown-0perator — COMMENTED** [review]
everything looks good, 
just small suggestions: consider heading hierarchical structure and as Sam mentioned before it is better to avoid inline styling

**Review by Unknown-0perator — DISMISSED** [review]

File: src/app/join-us/page.tsx [hunk 1]
```diff
@@ -1,32 +1,166 @@
...
+        </Typography>
+        <Typography variant="body2" paragraph>
+          Quis neque, eu et ipsum amet, vel et. Varius integer enim pellentesque
+          ornare pharetra faucibs arcu. Mauris blandit egestas nibh. eu et ipsum
+          amet, vel et. Varius integer enim pellentesque ornare pharetra faucibs
+          arcu. Mauris blandit egestas nibh.
+        </Typography>
+      </Box>
+      <Box sx={{ width: '40%', alignSelf: 'center' }}>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
```
Comment (CorreyL) [inline src/app/join-us/page.tsx:48]:
```suggestion
        <Box sx={{ display: 'flex', justifyContent: 'space-around' }}>
```

This is closer to the Figma design:

## Design

image

## Before

image

## After

image

Note the padding on the outer edges of the element

File: src/app/join-us/page.tsx [hunk 2]
```diff
@@ -1,32 +1,166 @@
...
+              </Box>
+            </Paper>
+            <Typography sx={{ marginLeft: '1em' }} variant="h5">
+              Tell Us About Yourself
+            </Typography>
+          </Box>
+          <Typography sx={{ marginLeft: '4em' }} variant="body2" paragraph>
+            Fill out our <u>2 minute form</u> so we can help match you with
+            opportunities
+          </Typography>
```
Comment (CorreyL) [inline src/app/join-us/page.tsx:88]: Admittedly this is possibly a nitpick, but I wonder for maintainability (i.e. if we ever want this to be mobile friendly), would it be possible to move some of the styling props here to the `.scss` file?

So for example:

```
             <Box
                sx={{
                  display: 'flex',
                  alignItems: 'center',
                  justifyContent: 'center',
                  height: '100%',
                }}
              >
```

```
<Typography sx={{ marginLeft: '1em' }} variant="h5">
```

```
<Typography sx={{ marginLeft: '4em' }} variant="body2" paragraph>
```

Could these CSS styles be moved to the `.scss` file, and then the appropriate classes called in the subsequent similar `Box` and `Typography` components below?

**Review by CorreyL — CHANGES_REQUESTED** [review]
Overall I think this is a good start to build off of! Some things we'll need to continue to work on in a future PR to more closely match the design include:

- Using the `Lato` font
- Include the usage of colors like in the design
image
- `Volunteers` and `Become A Partner` should be clickable (I believe?)

Just have some suggested code to match the design a bit more closely, but overall, looks like a good foundation to build off of!

**Comment by PooriaT** [comment]
Thanks, @CorreyL for your review. I made the changes based on your comments.

File: src/app/join-us/page.tsx [hunk 3]
```diff
@@ -1,32 +1,151 @@
...
+        </Typography>
+        <Typography variant="body2" paragraph>
+          Quis neque, eu et ipsum amet, vel et. Varius integer enim pellentesque
+          ornare pharetra faucibs arcu. Mauris blandit egestas nibh. eu et ipsum
+          amet, vel et. Varius integer enim pellentesque ornare pharetra faucibs
+          arcu. Mauris blandit egestas nibh.
+        </Typography>
+      </Box>
+      <Box className="box-size-40__center">
+        <Box className="display-flex__space-between">
```
Comment (CorreyL) [inline src/app/join-us/page.tsx:50]: I like how this was cleaned up to the SCSS, but in my last comment, I recommended `space-around` to more closely match the Design:

https://github.com/OpenBCca/openbc-web/pull/90#discussion_r1477135146

However, if `space-between` is the actual desired look, then we can leave it!

File: src/app/join-us/page.tsx [hunk 4]
```diff
@@ -1,32 +1,151 @@
...
+              <Box className="box-number">
+                <Typography>3</Typography>
+              </Box>
+            </Paper>
+            <Typography className="margin-left-1" variant="h5">
+              Have a question?
+            </Typography>
+          </Box>
+          <Typography className="margin-left-4" variant="body2" paragraph>
+            Find you answers in FAQ or email openbc.ca@gmail.com with any
```
Comment (CorreyL) [inline src/app/join-us/page.tsx:112]:
```suggestion
            Find you answers in FAQ or email{' '}
            <a href="mailto:&#x6f;&#x70;&#x65;&#x6e;&#x62;&#x63;&#x2e;&#x63;&#x61;&#x40;&#x67;&#x6d;&#x61;&#x69;&#x6c;&#x2e;&#x63;&#x6f;&#x6d;">
              openbc.ca AT gmail.com
            </a>{' '}
            with any
```

Few things I'm suggesting here:

1. Adding a `mailto:` link to make contacting us easier
2. Doing HTML character encoding on the e-mail address to minimize the risk of a scrapper adding our e-mail address to mailing/spam lists
https://stackoverflow.com/a/42369575
3. Replacing the `@` character with `AT` for similar reasons as 2

Seems to work as expected when testing my suggested code locally:

https://github.com/OpenBCca/openbc-web/assets/16601729/babb043d-003f-49ed-bbce-7d91459afe21

**Review by CorreyL — APPROVED** [review]
Not going to block the PR from getting merged, but I do have a couple small recommendations, but I'm happy to just add them in a future PR too

File: src/app/join-us/page.tsx (hunk 4 above)
Comment (PooriaT) [reply src/app/join-us/page.tsx:112]: That's great Corry.

**Review by CorreyL — APPROVED** [review]
Thanks for accepting my changes, looking good now!

**Pull request merged**